packages/shared/src/components/filters/AdvancedSettings.tsx

```diff
@@ -1,24 +1,68 @@
-import React, { ReactElement, useMemo } from 'react';
+import React, { ReactElement, useCallback, useMemo } from 'react';
 import useFeedSettings from '../../hooks/useFeedSettings';
 import { FilterSwitch } from './FilterSwitch';
 import { AdvancedSettingsGroup } from '../../graphql/feedSettings';
 import { useAdvancedSettings } from '../../hooks/feed';
+import useTagAndSource from '../../hooks/useTagAndSource';
+import { Origin } from '../../lib/analytics';
+import { Source } from '../../graphql/sources';
 
 const ADVANCED_SETTINGS_KEY = 'advancedSettings';
 
 function AdvancedSettingsFilter(): ReactElement {
-  const { advancedSettings, isLoading } = useFeedSettings();
+  const { advancedSettings, isLoading, feedSettings } = useFeedSettings();
   const { selectedSettings, onToggleSettings } = useAdvancedSettings();
+  const sourceList = useMemo(
+    () =>
+      advancedSettings?.filter(
+        ({ group }) => group === AdvancedSettingsGroup.ContentSource,
+      ) ?? [],
+    [advancedSettings],
+  );
   const settingsList = useMemo(
     () =>
       advancedSettings?.filter(
-        ({ group }) => group === AdvancedSettingsGroup.Advanced,
+        ({ group }) => group === AdvancedSettingsGroup.ContentCuration,
       ) ?? [],
     [advancedSettings],
   );
 
+  const isBlocked = useCallback(
```

Comment:
this looks like a `boolean`, mabe something like `checkSourceBlocked`?

Reply:
Yeah is a boolean indeed, will do :)